lib/server.js

```diff
@@ -8,22 +8,27 @@ module.exports = {
 
   createServer (config) {
     const server = new Hapi.Server()
-    server.connection({
+    const defaultOptions = {
       host: config.web.host,
       port: config.web.port,
       routes: {
         files: {
           relativeTo: config.main.paths.root
         }
       }
-    })
+    }
+
+    server.connection(_.defaultsDeep(config.web.options || {}, defaultOptions))
```

Comment:
We should make sure `config.web.options` is set to the value of `defaultOptions` even if it doesn't exist. That way the values in `config.web.options` will be exactly the same as whatever Hapi uses to initialize